### PR TITLE
wifi-subsys: added functionality to send messages to server from uart

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

modifications were made to be able to wifi-subsys so that when sensor data arrive they are sent to the remote server changes were also added to the https client to send cbor values

### CODE REVIEW

File: wifi-subsys/components/uart/src/subsys_uart.c [hunk 1]
```diff
@@ -85,9 +93,24 @@ int sendData(const char *logName, char *data) {
...
+    ESP_LOGI(__func__, "Read sensor data: '%s' and len is: %d ", message, len);
+    decode(values, "temp", &val_received, len);
+
+    decode(values, "node_id", &val_received, len);
+    ESP_LOGI(__func__, "%d ", val_received);
+
+    http_request_t request;
+    request.method = HTTP_METHOD_POST;
+    request.content_type = HTTPS_CONTENT_CBOR;
+    memcpy(request.url, "https://cloud.mesh4all.org/?a=1&b=2&c=3",
```
Comment (RocioRojas) [inline wifi-subsys/components/uart/src/subsys_uart.c:106]: Change -> a=1&b=2&c=3

File: wifi-subsys/components/uart/src/subsys_uart.c [hunk 2]
```diff
@@ -85,9 +93,24 @@ int sendData(const char *logName, char *data) {
...
+           sizeof("https://cloud.mesh4all.org/?a=1&b=2&c=3"));
```
Comment (RocioRojas) [inline wifi-subsys/components/uart/src/subsys_uart.c:107]: change https://cloud.mesh4all.org/?a=1&b=2&c=3

File: wifi-subsys/components/protocols/src/httpsclient.h [hunk 3]
```diff
@@ -38,6 +38,7 @@ extern "C" {
 #define MAX_HTTP_OUTPUT_BUFFER 1024 /*!< maximum size to send */
 
 #define HTTPS_CONTENT_JSON "application/json"
+#define HTTPS_CONTENT_CBOR "application/cbor"
```
Comment (eduazocar) [inline wifi-subsys/components/protocols/src/httpsclient.h:41]: Remove this, this it's not working in https_protocol component

File: wifi-subsys/components/uart/src/subsys_uart.c [hunk 4]
```diff
@@ -31,6 +31,8 @@
 #include "at_handler.h"
 #include "wifi.h"
 #include "default_params.h"
+#include "serialization.h"
+#include "httpsclient.h"
```
Comment (eduazocar) [inline wifi-subsys/components/uart/src/subsys_uart.c:35]: In the module or component file, if you are including others modules, you are breaking their independency, I refers to use uart_at, you will need to load component/tools and component/protocols

**Review by eduazocar — CHANGES_REQUESTED** [review]

**Comment by RocioRojas** [comment]
Good job!, only one change, the name of PR.

**Review by eduazocar — APPROVED** [review]

**Review by RocioRojas — APPROVED** [review]

**Pull request merged**